
[25-Jul-2023 18:06:01 America/Argentina/Buenos_Aires] PHP Fatal error:  Uncaught Error: Class "Sentry\Laravel\Integration" not found in /home/customer/www/dev.msklatam.com/public_html/msk-laravel/app/Exceptions/Handler.php:46
Stack trace:
#0 [internal function]: App\Exceptions\Handler->App\Exceptions\{closure}(Object(Error))
#1 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/ReportableHandler.php(45): call_user_func(Object(Closure), Object(Error))
#2 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(256): Illuminate\Foundation\Exceptions\ReportableHandler->__invoke(Object(Error))
#3 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(182): Illuminate\Foundation\Exceptions\Handler->report(Object(Error))
#4 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(254): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#5 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#6 {main}
  thrown in /home/customer/www/dev.msklatam.com/public_html/msk-laravel/app/Exceptions/Handler.php on line 46
[25-Jul-2023 18:07:00 America/Argentina/Buenos_Aires] PHP Fatal error:  Uncaught Error: Class "Sentry\Laravel\Integration" not found in /home/customer/www/dev.msklatam.com/public_html/msk-laravel/app/Exceptions/Handler.php:46
Stack trace:
#0 [internal function]: App\Exceptions\Handler->App\Exceptions\{closure}(Object(Error))
#1 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/ReportableHandler.php(45): call_user_func(Object(Closure), Object(Error))
#2 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(256): Illuminate\Foundation\Exceptions\ReportableHandler->__invoke(Object(Error))
#3 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(182): Illuminate\Foundation\Exceptions\Handler->report(Object(Error))
#4 /home/customer/www/dev.msklatam.com/public_html/msk-laravel/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(254): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Error))
#5 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(Error))
#6 {main}
  thrown in /home/customer/www/dev.msklatam.com/public_html/msk-laravel/app/Exceptions/Handler.php on line 46
